agent/main.go

```diff
@@ -40,8 +40,7 @@ const (
 	defaultWCPollURL         = "https://{{.WCHostname}}/k8s.yaml" +
 		"?k8s-version={{.KubernetesVersion}}&t={{.Token}}&omit-support-info=true" +
 		"{{if .FluxConfig}}" +
-		"&git-label={{.FluxConfig.GitLabel}}&git-url={{.FluxConfig.GitURL}}" +
-		"&git-path={{.FluxConfig.GitPath}}&git-branch={{.FluxConfig.GitBranch}}" +
+		"&{{.FluxConfig.AsQueryParams}}" +
```

Comment:
This is maybe not working? Investigating after lunch! 🥞

Reply:
Oh no it is the getConfig w/ bool args: 

`"Failed getting existing flux config: bool flag `--git-ci-skip' cannot have an argument"`